src/main/java/net/datafaker/providers/base/Finance.java

```diff
@@ -115,18 +126,34 @@ public String bic() {
         return faker.regexify("([A-Z]){4}([A-Z]){2}([0-9A-Z]){2}([0-9A-Z]{3})?");
     }
 
+    /**
+     * Generates a random IBAN (International Bank Account Number).
+     * The country code is chosen randomly from the supported countries.
+     * The generated IBAN will pass the checksum test.
+     * @return a valid IBAN
+     */
     public String iban() {
         List<String> countryCodes = new ArrayList<>(countryCodeToBasicBankAccountNumberPattern.keySet());
         String randomCountryCode = countryCodes.get(faker.random().nextInt(countryCodes.size()));
         return iban(randomCountryCode);
     }
 
+    /** Generates an IBAN for the given country code.
+     * The generated IBAN will pass the checksum test.
+     * @param countryCode the 2-letter country code for the IBAN.
+     * @return a valid IBAN
+     */
```

Comment:
Though it's minor, usually the first line contains only "/**" characters. And the text starts from the second line:

```
    /**
     * Generates an IBAN for the given country code.
     * The generated IBAN will pass the checksum test.
     * @param countryCode the 2-letter country code for the IBAN.
     * @return a valid IBAN
     */
```